### ISSUE TITLE
abort trap: 6

### ISSUE THREAD

**Issue by brockmt** [heaslon/Easy-Leaf-Area#3]
Hi;  I have been using ELA for the past several years;  super useful!  However, when I try to start the app via terminal I get the following:
  
  calib file not found
  Set to default arabidopsis values 
  macOS 11 (1107) or later required, have instead 11 (1106) !
  Abort trap: 6

(I never built a calibration file but the default to arabidopsis values has always worked for me).

I'm currently running: 11.7 on a Mac mini (2018).  I'm curious if A) your still keeping an eye on this repository? and B) if there is a simple fix for this error?

Thanks in advance for any help you could provide.
Best
Marc

**Comment by heaslon** [comment]
Hi Marc,
I can still get the distributable version to run on my mac (Catalina).  If
you are running the python script, maybe try running the distributable
version instead.  Let me know if that helps or if it doesn't.
[image: Screen Shot 2022-12-07 at 11.36.23 AM.png]

I don't have time to work on ELA this year, but I may have some time next
year to update it.
Best,
Hsien

**Comment by brockmt** [comment]
Hi Hsien,
Thanks for your reply.  I'm using Big Sur on intel Mac mini, just updated to 11.7.1 today and am still getting the abort trap 6 notification (which I noticed started happening under 11.7).  I don't have time machine active or I"d revert back and see where things broke down.   I also downloaded the Mac version a second time (didn't think it would help but....) and got the same error message.  

I finally just swapped to the PC version on a different computer so can solve the issue that way.   Thanks again.
Best,
Marc
